travvy/src/components/Profile.vue

```diff
@@ -1,42 +1,54 @@
 <template>  
   <div class="Profile">
      <div class="container">
-        <h2>Profile Information</h2>
-    <label>Email: </label>
-    <input type="text" class="info" v-model="email">
-    <br>
-    <br>
-     <label>Gender: </label>
-    <input type="text" select class="info" v-model="gender">
-    <br>
-    <br>
-    <label>Age: </label>
-    <input type="text" class="info" v-model="age" > 
-    <br>
-    <br>
-    <label>Preferred activity: </label>
-    <input type="text" class="info" v-model="preferred_activity_type"> 
-    <br>
-    <br>
-    <label>Instagram handle: </label>
-    <input type="text" class="info" v-model="instagram_username"> 
-    <br>
-    <br>
-    <ul>
-      <li><a href="#">Edit profile</a></li>
-    </ul>
+        <h2> Hello {{  getUser().Name }}{{ this.$store.state.name}},</h2>
```

Comment:
Amazing! Storing these fields within the input placeholder will allow us to implement the 'update profile settings' very efficiently

Reply:
Yes definitely. You will just need to make sure you update things in both the database AND the store for the website to function correctly.